src/test/java/org/opensearch/knn/bwc/KNNBackwardsCompatibilityIT.java

```diff
@@ -82,6 +93,7 @@ private ClusterType getClusterType(){
     @SuppressWarnings("unchecked")
     public void testBackwardsCompatibility() throws Exception {
         String uri = getUri(getClusterType());
+        RecallTestsIT recallTestsIT = new RecallTestsIT();
```

Comment:
I am not sure whether you can use one test method in another test

Reply:
refactored the code